Eatery/Controllers/BRB/BRBAccountViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  BRBAccountViewController.swift
+//  Eatery
+//
+//  Created by William Ma on 9/1/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: -
+
+class BRBAccountViewController: UIViewController {
+    
+    private let account: BRBAccount
+    
+    private var tableView: UITableView!
+    
+    init(account: BRBAccount) {
+        self.account = account
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.register(BRBBalanceTableViewCell.self, forCellReuseIdentifier: "balance")
+        tableView.register(BRBHistoryTableViewCell.self, forCellReuseIdentifier: "history")
+        tableView.allowsSelection = false
+        tableView.separatorColor = .wash
+        tableView.dataSource = self
+        tableView.delegate = self
+        view.addSubview(tableView)
+        tableView.snp.makeConstraints { make in
+            make.top.edges.equalToSuperview()
+        }
+    }
+
+}
+
+// MARK: -
+
+extension BRBAccountViewController: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return 4
+        }
+        return account.history.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.section == 0 {
+            let cell = tableView.dequeueReusableCell(withIdentifier: "balance") as! BRBBalanceTableViewCell
```

Comment:
Can we make the cell identifiers a constant since we use it in more than one place?